homeassistant/components/bayesian/binary_sensor.py

```diff
@@ -21,6 +21,7 @@
 from homeassistant.helpers.event import async_track_state_change
 
 ATTR_OBSERVATIONS = "observations"
+ATTR_ENTITIES = "observed_entities"
```

Comment:
Don't make it this. Isntead use `ATTR_ENTITY_ID` from `homeassistant.const`. That's the one that is used by other integrations too.

Reply:
Done